sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -362,13 +362,51 @@ class SparkSqlParserSuite extends AnalysisTest {
         Seq(("TOK_TABLEROWFORMATFIELD", ","),
           ("TOK_TABLEROWFORMATCOLLITEMS", "#"),
           ("TOK_TABLEROWFORMATMAPKEYS", "@"),
-          ("TOK_TABLEROWFORMATLINES", "\n"),
-          ("TOK_TABLEROWFORMATNULL", "null")),
+          ("TOK_TABLEROWFORMATNULL", "null"),
+          ("TOK_TABLEROWFORMATLINES", "\n")),
```

Comment:
is it necessary to switch the order here?

Reply:
> is it necessary to switch the order here?

Yea, follow change from SparkSqlParser.